contributing/release-schedule.md

```diff
@@ -121,3 +113,22 @@ In determining how often we should push new versions of AMP to everyone, we have
 - the desire to get new features out quickly
 
 After considering all of these factors we have arrived at the 1-2 week push cycle.  Thus far we have found this to be a reasonable compromise, but we will continue to evaluate all of these factors and may make changes in the future.
+
+## Release freezes
+
+There are occasions when we will skip a release of AMP to production, known as a release freeze.
+
+If a one week release freeze is announced for Week N:
+- The previous week's release build remains in canary/RC for an extra week, i.e. the release cut in Week N-1 is not pushed to production in Week N as would normally be the case.  Instead, it will be pushed to production in Week N+1.
+- A new release build is *not* made in the production freeze week (Week N).
+- The normal schedule will resume in Week N+1, i.e. a canary is cut in Week N+1 and pushed to production in Week N+2.
+
+A release freeze will happen due to:
+
+- Times when there are not enough people available to push the AMP release to production and monitor it.  Currently most of the people performing AMP releases are based in the United States, so this will usually be the weeks of the major US holidays of Independence Day (4 July), Thanksgiving (4th Thursday in November), Christmas (December 25) and New Year's Eve/Day (December 31/January 1).
```

Comment:
Nit: "July 4" (consistent with the other dates) and replace "4th Thursday" to "fourth Thursday" to avoid confusing readers with numbers as days

Reply:
thanks for the catch!